ad-code-manager.php

```diff
@@ -272,52 +272,52 @@ function update_options( $new_options ) {
 	 */
 	function handle_admin_action() {
 
-		if ( !wp_verify_nonce( $_REQUEST['nonce'], 'acm-admin-action' ) )
-			wp_die( __( 'Doing something fishy, eh?', 'ad-code-manager' ) );
+		if ( isset( $_REQUEST['nonce'] ) && ! wp_verify_nonce( sanitize_key( $_REQUEST['nonce'] ), 'acm-admin-action' ) ) // Input var okay.
```

Comment:
We shouldn't use REQUEST but rather use either POST or GET as applicable (REQUEST encompasses COOKIE and if a nonce is for some reason incorrectly stored in a cookie it would allow CSRF since the cookie would be sent along)

Reply:
I think $_REQUEST was used because the switch routine handles both POST and GET, depending on the conditions.  I think I found a better way to deal with this and still make VIPCS happy.  ;-)